src/frontend/src/lib/utils/convert.utils.ts

```diff
@@ -0,0 +1,41 @@
+import { ZERO } from '$lib/constants/app.constants';
+import type { ConvertAmountErrorType } from '$lib/types/convert';
+import { formatToken } from '$lib/utils/format.utils';
+import { isNullish, nonNullish } from '@dfinity/utils';
+import { BigNumber } from '@ethersproject/bignumber';
+import { Utils } from 'alchemy-sdk';
+
+export const validateConvertAmount = ({
+	userAmount,
+	decimals,
+	balance,
+	totalFee
+}: {
+	userAmount: BigNumber;
+	decimals: number;
+	balance?: BigNumber;
+	totalFee?: bigint;
+}): ConvertAmountErrorType => {
+	if (isNullish(totalFee)) {
+		return;
```

Comment:
I saw that in the code we do the same elsewhere, but I don't remember why? is it possible that the validation comes in another component when the fee is nullish?

Reply:
That's a good point. After thinking about it, I think it makes sense to validate `insufficient-funds` case even if `totalFee` is nullish. Therefore, I removed this check and adjusted the tests.